src/utils.ts

```diff
@@ -24,11 +23,6 @@ async function callStrategy(
   ) {
     return {};
   }
-
-  if (!_strategies.hasOwnProperty(strategy.name)) {
-    throw new Error(`Invalid strategy: ${strategy.name}`);
-  }
```

Comment:
we may need this, it may throw undefined error. or should add a check directly on `getScoresDirect`?